pallets/thea/src/lib.rs

```diff
@@ -446,7 +446,6 @@ pub mod pallet {
 		) -> DispatchResult {
 			let user = ensure_signed(origin)?;
 			// Put a soft limit of size of beneficiary vector to avoid spam
-			ensure!(beneficiary.len() <= 100, Error::<T>::BeneficiaryTooLong);
```

Comment:
Comment should be moved in `do_withdraw` method

Reply:
done